### DISCUSSION TITLE
CrossGridder 1.2 is here

### DISCUSSION THREAD

**Discussion by CrossGridder** [apohl79/audiogridder#835] [Ideas]
After a month of testing I am proud to announce the first stable version of CrossGridder. Check my repositories for download and an in-depth PDF guide with pictures and solutions to most problems reported here and for which there was no solution yet. 

After reading many discussions from AG users and Andreas, I am quite sure that the reading of the PDF and the use of CrossGridder could help the development of Audiogridder itself. The creation of a window environment reported in issues is just an example, I am trying to answer to all posts that could find an answer in CrossGridder. 

WHAT IS CROSSGRIDDER

. CROSSGRIDDER is an AudioGridderServer.exe wrapped into a Crossover 21 bottle, 
and its able to do things that AG server can't yet and some others that,
due to its nature, simply can't be implemented.

. Follow this whole guide to take advantage of every CrossGridder's feature. 
I wrote it with one goal in mind:
provide a comprehensive step by step guide full of details, 
letting you unleash its full power and capabilities.

. Being CrossGridder a CrossOver bottle means that VSTs will 	
work pretty much like native ones. Disabled Sidechain is a current  	
AudioGridder server limitation.

. Since version 1.2, it comes in different versions to avoid 	
importing issues, however, if you have problems importing one, 	
try the other:

	. CG_1.2_Legacy (Mojave>Catalina/Linux)
	. CrossGridder_1.2 (Catalina > Monterey/Linux)

. By default It contains freeware softwares. It is up to you to 	
add more VSTs and build your own dream bottle.

. CrossGridder is free and will always be, so please consider a 	
donation to help me keep this project going on and 	
be able to technically assist you. 	

. If none can be imported in your system, please send an email to CrossGridder@gmail.com
with your system's details and a brief description of the problem.
WHAT CROSSGRIDDER CAN DO FOR YOU!

.. Be a Linux Server (not available yet as an official AG server):
. Use VSTs on a Linux machine setup on any Linux DAW that supports the AG plugin. 
Give a chance to your old PC/Mac installing a Linux distro. 
If you have an Intel Core processor and a not too old card
(something like HD 4000 should work), you should be good to go.

.. Use VSTs on ProTools with the AG AAX plugin

.. Create as many Servers as your system can handle
. To overcome AG server limitation of 1 GUI per server.
. Stop dealing with 2 desktops on 1 screen to see all the GUIs.
. Stop freezing of a whole chain of plugins if one server freezes. 

.. Use it a as sandboxed environment for VSTs trial:
. Do not install the native trial plugin in your OS, install Windows version in CrossGridder.
. When you decide that the new plugin is worth (or not!) to be part of your main system, 
delete the bottle and install it (or not!) your OS native plugin.
You can have as many copies of CrossGridder as you wish, 
so you can even sandbox the sandbox, if you know what I mean :)

.. Or as a replacement for virtual machines on macOS and Linux:
. Plugins will run pretty much like natives. No CPUs dedicated to virtual machines, 
less disk space and ram wasted and only one screen to deal with, when used on a one machine setup.

**Comment by apohl79** [comment]
Hey all,

I'm just realizing this thread now and I'm a bit shocked about the direction it took. I will lock the thread now, not sure if I should delete the convo, as it is pretty negative..

Anyways, just to make that clear again, I have chosen the MIT license for AudioGridder by purpose, as it basically allows anybody to use the software freely with no limitations. This includes new projects or as a base for for something else etc. So my "permission" was never required.

That said - and sorry if this sounds lecturing, I know we are all grown ups - no one should offend others and on the other hand, taking things personally if people don't like other peoples work does not help either.

I'm very grateful about AG helping others with their workflows, so increasing it's compatibility especially with a guide seems like a good idea. 

Happy Saint Nicholas!
Andreas
